src/requestsV.py

```diff
@@ -37,25 +37,35 @@ def __init__(self, version, log, Error):
     @staticmethod
     def check_version(version, copy_run_update_script):
         # checking for latest release
-        r = requests.get("https://api.github.com/repos/zayKenyon/VALORANT-rank-yoinker/releases")
-        json_data = r.json()
-        release_version = json_data[0]["tag_name"]  # get release version
-        for asset in json_data[0]["assets"]:
-            if "zip" in asset["content_type"]:
-                    link = asset["browser_download_url"]  # link for the latest release
-                    break
-        if float(release_version) > float(version):
-            print(f"New version available! {link}")
-            if sys.argv[0][-3:] == "exe":
-                while True:
-                    update_now = input("Do you want to update now? (Y/n): ")
-                    if update_now.lower() == "n" or update_now.lower() == "no":
-                        return
-                    elif update_now.lower() == "y" or update_now.lower() == "yes" or update_now == "":
-                        copy_run_update_script(link)
-                        os._exit(1)
-                    else:
-                        print('Invalid input please response with "yes" or "no" ("y", "n") or press enter to update')
+        try:
+            r = requests.get("https://api.github.com/repos/zayKenyon/VALORANT-rank-yoinker/releases")
+        except requests.exceptions.RequestException:
+            print(color("[ WARNING ] Unable to check for updates - skipping...", fore=(255, 165, 0)))
```

Comment:
Please reformat the message:
[WARNING] Unable to check for updates - skipping...

Remove the redundant spaces in square brackets, in other print's too.